console_bot/birthdays_per_week.py

```diff
@@ -34,7 +34,7 @@ def get_birthdays_in_next_days(users, days):
             result_dates(result, user_name, birthday_this_year)
     if not result:
         return f"No birthdays in next {days} days"
-    return "\n".join([f"{date}: {(',').join(people)}" for date, people in result.items()])
+    return "\n".join([f"{date}: {(",").join(people)}" for date, people in result.items()])
 
 def get_today_birthday(users):
```

Comment:
the changes are needed in return statement of this function

Reply:
Now should be good